source/run-agg-pipelines.txt

```diff
@@ -9,5 +9,140 @@ Run Aggregation Pipelines
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
+
+You can run :manual:`aggregation pipelines </aggregation>` on your
+collections in |vsce|. Aggregation pipelines consist of
+:manual:`stages </reference/operator/aggregation-pipeline/>` which
```

Comment:
How about one of the following wording: 
Aggregation pipelines consist of
:manual:`stages </reference/operator/aggregation-pipeline/>`, which ...
OR
Aggregation pipelines consist of
:manual:`stages </reference/operator/aggregation-pipeline/>` that ...

Reply:
Went with your second suggestion.